### ISSUE TITLE
Auxiliary routes not taken into account on application start ?

### ISSUE THREAD

**Issue by stanislasdrg** [angular/angular#12617]
**I'm submitting a ...**  (check one with "x")

```
[ x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
I'm resolving to posting here, as I couldn't get any answer on SO, and it looks an Angular bug :
It seems that auxiliary routes aren't taken into account on application start.
[Cf. Related plunk](http://plnkr.co/edit/gQKpvFkXk2Ne8I1GPNAR?p=info)
[Related SO question](http://stackoverflow.com/questions/40288993/angular-2-how-to-load-auxiliary-route-on-application-start) 

I'm trying to load an auxiliary route on app start. Everything works as expected when using a RouterLink, or when entering URL manually. However I can't get the route to load straight away.
I have no way to get around this, as I must use routes in order to resolve API data for my components.

**Expected behavior**
The auxiliary route should load on application start.

**Minimal reproduction of the problem with instructions**
Please Fork [this plunk](http://stackoverflow.com/questions/40288993/angular-2-how-to-load-auxiliary-route-on-application-start). Removing the auxiliary route and the named router outlet results in the app displaying correctly.

**What is the motivation / use case for changing the behavior?**
Using resolvers in child components on application start.

**Please tell us about your environment:**
MBP under MacOs Sierra, Atom editor, Angular2 Typescript with Webpack (Angular-cli).
Angular version : 2.1.2
Angular router version : 3.1.0
- **Browser:** 
  Chrome 54, Safari 10
- **Language:** 
  Typescript 2, Es6 
- **Node (for AoT issues):** `node --version` =   6.0.42

**Comment by vsavkin** [comment]
The issue is that we always require a primary route to be present. The following will fix the issue:
https://github.com/angular/angular/issues/12133

**Comment by stanislasdrg** [comment]
Thank you very much, it does work that way.

**Comment by vicb** [comment]
fixed by #12133
